pipeline_dp/aggregate_params.py

```diff
@@ -131,6 +131,9 @@ class AggregateParams:
          max_contributions_per_partition. This option can be used if the dataset
          does not contain any identifiers that can be used to enforce
          contribution bounds automatically.
+        public_partitions_already_filtered: if true, it is assumed that records
+         with partition_key which are not in public_partitions are already
+         removed from the dataset. It applies only for public partitions.
```

Comment:
"It applies only for public partitions." -> "The flag can only be used with public partitions"

Reply:
Thanks